ManatiCR/manatime
# Manatime-21: Clients listing view.

**Steps to test**
- See if the clients listing view is available.
- Add a couple of clients nodes.
- Confirm wercker build is sucesfully.

**DoD**
- As an administrator I want to go /clients and get a table with a list of Client Names.
- In that view, I need to have direct links to edit clients
- I need to have 2 views (/clients/active and /clients/inactive and by default /clients go to /clients/active)

[Issue Link](https://github.com/ManatiCR/manatime/issues/21)
